src/chapters/aspdotnet-controllers-and-routes/controllers-attributes.rst

```diff
@@ -0,0 +1,115 @@
+Streamling Your Code with Attribute Routing
```

Comment:
Modified the project to only use one method, ``Welcome()``, but still left the part about a method responding to different requests at the similar routes to this page, which also has a different name. I think the other pages got a lot longer so I thought it might be easier to go over that concept in a separate dedicated zone.